python/geom.py

```diff
@@ -178,6 +287,54 @@ def __init__(self, epsilon_diag=Vector3(1, 1, 1),
                  H_chi2=None,
                  H_chi3=None,
                  valid_freq_range=FreqRange(min=-mp.inf, max=mp.inf)):
+        """
+        Creates a `Measure` object.
```

Comment:
Medium?

Reply:
Oops. I kept making that switch in my head, not sure why I had "Measure" on my mind.